lib/cli/commands/listtrades.ts

```diff
@@ -0,0 +1,45 @@
+import { Arguments } from 'yargs';
+import { callback, loadXudClient } from '../command';
+import { ListTradesRequest, ListTradesResponse, Trade } from '../../proto/xudrpc_pb';
+import Table, { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+
+const HEADERS = [
+  colors.blue('rhash'),
+  colors.blue('trade quantity'),
```

Comment:
These should be capitalized (such as "Trade quantity") to be consistent with the tables in other calls.